src/components/ContributePayment.js

```diff
@@ -273,6 +273,7 @@ class ContributePayment extends React.Component {
                 <Box my={3}>
                   <NewCreditCardForm
                     name="newCreditCardInfo"
+                    profileType={this.props.collective.type}
```

Comment:
As per propTypes, `collective` can be null:

```es6
  /**
   * An optional collective to get payment methods from. If used at the same time as
   * `paymentMethods` it will merge both lists and filter uniques using their ids.
   */
  collective: PropTypes.object,
```

This is the case for the styleguide example, where we don't provide a collective: https://github.com/opencollective/opencollective-frontend/blob/c54ef86ab26edff0dfdcb4df8b62f2243f1ab77e/styleguide/examples/ContributePayment.md

So we should do:
```suggestion
                    profileType={get(this.props.collective), 'type')}
```